docs/emr_deployment.md

```diff
@@ -0,0 +1,126 @@
+---
+id: emr_deployment
+title: GeoTrellis EMR deployment with GDAL
+sidebar_label: GeoTrellis EMR deployment with GDAL
+---
+
+GeoTrellis can be used with _EMR 6_. However, if there is a need in GDAL bindings usage there can be some certain complexities.
+This section describes the entire _EMR 6_ deployment and GDAL installation in a way that would satisfy GeoTrellis GDAL requirements.
+
+Starting _GeoTrellis_ `3.4.2`, _GeoTrellis_ is compatible only with the GDAL`3.1 +` versions.
+
+## Installing GDAL on EMR
+
+The EMR installation conatans of the following steps:
+
+1. Install _GDAL_
+2. Set Spark `LD_LIBRARY_PATH`. In this case `GeoTrellis GDAL` would be able to use the installed `GDAL`.
+
+You can also check out [GeoTrellis Spark template](https://github.com/geotrellis/geotrellis-spark-job.g8) that is properly configured for the EMR jobs.
+
+### EMR Bootsrap script (GDAL installation on EMR)
+
+The EMR bootstrap script with all appropriate settings is located under the following S3 URI: [s3://geotrellis-test/emr-gdal/bootstrap.sh](s3://geotrellis-test/emr-gdal/bootstrap.sh).
+It installs GDAL 3.1.2 (by default) through conda and sets all appropriate env variables in case 
```

Comment:
Isn't including the phrase `(by default)` redundant here because the only installation option is with conda?

Perhaps, again simplifying:

> It installs GDAL 3.1.2 with conda and configures env variables for use with `spark-shell` on the EMR nodes.

Reply:
@CloudNiner conda by default installs `latest`; do you know how to better frame that this bootstrap scripts uses 3.1.2 as the default gdal version?

Reply:
Ah. I believe you can do `conda install <pkg>=<version>`